### PR TITLE
Contributions tests now returns the ledger-info state data

### PR DESCRIPTION
**Pull request by jasonrsadler**
Previously return only the transactions object, the contributions test now returns the data from ['ledger', 'info'] part of state.

### CODE REVIEW

File: js/contribution.js [hunk 1]
```diff
@@ -157,8 +149,7 @@ class JS extends Contribution {
     this.fuzzPublishers()
     this.state = ledgerState.setInfoProp(this.state, 'reconcileStamp', 1528948800000)
     this.ledger.run(this.state, 0)
-    this.staticizeBallots()
-    return ledgerState.getInfoProp(this.state, 'transactions') || []
+    return this.state.getIn(['ledger', 'info'])
```
Comment (ryanml) [inline js/contribution.js]: There is this:

https://github.com/brave-intl/bat-ledger-spec/blob/master/js/spec.js#L102

Inside spec.js, which this class inherits, could we maybe use this/refactor current method perhaps?

**Review by ryanml — COMMENTED** [review]
Added a single comment, else, lgtm :)

File: js/contribution.js (hunk 1 above)
Comment (jasonrsadler) [reply js/contribution.js]: Yes. Can use it as-is. Done.

**Review by ryanml — APPROVED** [review]
lgtm!

**Pull request merged**